.gitignore

```diff
@@ -1,4 +1,7 @@
 /.pytest_cache
 /.idea
 /chrome*
-/__pycache__
+__pycache__/
```

Comment:
AFAIK, the first line would be enough to ignore all directories in project by name

also, https://stackoverflow.com/questions/1470572/ignoring-any-bin-directory-on-a-git-project confirming it